start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java

```diff
@@ -43,8 +44,40 @@ private RootGradleBuild createGradleBuild(String name, BuildItemResolver buildIt
 
     @Bean
     public BuildCustomizer<RootGradleBuild> addDockerVersion(ProjectDescription projectDescription) {
-        return (build) -> build.properties().property("alfrescoVersion", quote(projectDescription.getPlatformVersion().toString()));
-
+        String alfrescoVersion;
+        String alfrescoArtifactId;
+        String dockerRegistry;
+        String alfrescoDockerImage;
+        Version version = projectDescription.getPlatformVersion();
+        // TODO until https://github.com/xenit-eu/start.xenit.eu/issues/23 is resolved, just list translations
+        switch (version.toString()) {
+            case "5.2.5":
+                alfrescoVersion = "5.2.5";
+                alfrescoArtifactId = "alfresco-enterprise";
+                dockerRegistry = "hub.xenit.eu/alfresco-enterprise";
+                alfrescoDockerImage = "alfresco-enterprise";
+                break;
+            case "5.2.0.g":
+                alfrescoVersion = "5.2.g";
+                alfrescoArtifactId = "alfresco-platform";
+                dockerRegistry = "xeniteu";
+                alfrescoDockerImage = "alfresco-repository-community";
+                break;
+            case "6.1.2.ga":
+                alfrescoVersion = "6.1.2-ga";
+                alfrescoArtifactId = "content-services-community";
+                dockerRegistry = "xeniteu";
+                alfrescoDockerImage = "alfresco-repository-community";
+                break;
+            default:
+                throw new IllegalArgumentException(version.toString() + " is not a supported version for now");
+        }
+        return (build) -> {
+            build.properties().property("alfrescoVersion", quote(alfrescoVersion));
```

Comment:
only `alfrescoVersion` should be in the gradle ext properties
all the other changes should be inlined